syft/pkg/cataloger/internal/cpegenerate/generate.go

```diff
@@ -62,16 +62,13 @@ func GetIndexedDictionary() (_ *dictionary.Indexed, err error) {
 
 func FromDictionaryFind(p pkg.Package) ([]cpe.CPE, bool) {
 	dict, err := GetIndexedDictionary()
-	parsedCPEs := []cpe.CPE{}
 	if err != nil {
 		log.Debugf("CPE dictionary lookup not available: %+v", err)
-		return parsedCPEs, false
+		return []cpe.CPE{}, false
```

Comment:
nit: this can be `nil`, I see that it was allocated to begin with, but since your changing much of this do we still need to allocate? (we probably never did, but I haven't checked)